bin/start.js

```diff
@@ -1,8 +1,10 @@
+'use strict'
+
+const resolveEntry = require('../lib/resolve-entry')
```

Comment:
@yoshuawuyts Are you okay with this? The current project setup seems to suggest you'd like to keep together in very flat organisation.